examples/bidspath.py

```diff
@@ -0,0 +1,158 @@
+"""
+===========================
+An introduction to BIDSPath
```

Comment:
aren't the examples ordered in terms of difficulty?

Reply:
I wasn't sure where to put this one, so I simply put it at the very end. Any suggestions on how to make it better are of course appreciated!

Reply:
do we still need 01. with this tutorial? Are they duplicating?

Reply:
> Are they duplicating?

I'm not sure what you mean by that..?

Reply:
I think @jasmainak's point here is to scrap our current 01. example ("Creating BIDS-compatible folder names and filenames") and expand the bidspath example to cover everything from that old 01. example

--> and then make the bidspath example the new 01. example, which would be fitting, because it's now becoming the center piece of mne-bids.

I would be in favor of that.

Reply:
I will give this a shot

Reply:
Thanks for expanding the thoughts in my mind @sappelhoff :)